pkg/system/phase4_configuring.go

```diff
@@ -682,6 +685,47 @@ func (r *Reconciler) RegisterToCluster() error {
 	return r.NBClient.RegisterToCluster()
 }
 
+func (r *Reconciler) ReconcileDefaultNamespaceStore() error {
```

Comment:
should require description comment to be exported

Reply:
Yes, will do that